src/dropdownVersion.js

```diff
@@ -0,0 +1 @@
+export default '2.9.9'
```

Comment:
Should these end with new line characters?

Reply:
These files are generated by the versionWriter.js in the auro-library. We can certainly look at doing an update to the script that that adds a new line. 

It has no impact on this repo and shouldn't hold up this release.